h2o-algos/src/main/java/hex/ensemble/Metalearners.java

```diff
@@ -134,7 +133,9 @@ protected void validateParams(Model.Parameters parms) {
             super.validateParams(parms);
 
             // Check if distribution family is supported and if not pick a basic one
-            if (!ArrayUtils.contains(_supportedDistributionFamilies, parms._distribution)) {
+            ModelBuilder mb = ModelBuilder.make(parms);
+            mb.init(false);
+            if (mb.error_count() > 0) {
```

Comment:
how do you know that those errors where distribution related? psychic?

Reply:
Fixed, thank you!